plugins/gif.rb

```diff
@@ -48,4 +43,16 @@ def on_message(message, matches)
 
     gifs.sample
   end
+
+  # unescape google urls with octal (!?) escapes into url-encoded hex equivalents
+  def unescape(url)
```

Comment:
I moved some methods used across plugins—e.g. [wallpaper](https://github.com/markolson/linkbot/blob/master/lib/linkbot/plugin.rb#L110)—to the base Plugin class if you wanted to define this method once there.